code/game/objects/items/devices/scanners.dm

```diff
@@ -131,227 +131,211 @@ SLIME SCANNER
+						if(H.mind)
+							temp_message += " <span class='danger'>Warning: subject may be possesed.</span>"
+						else
+							temp_message += " <span class='notice'>Subject appears to be astrally projecting.</span>"
+
+
+			//LIVER
+			if(istype(O, /obj/item/organ/liver))
+				var/obj/item/organ/liver/L = O
+				if(H.undergoing_liver_failure() && H.stat != DEAD) //might be depreciated
+					temp_message += "<span class='danger'>Subject is suffering from liver failure: Apply Corazone and begin a liver transplant immediately!</span>"
+				if(L.swelling > 20)
+					temp_message += " <span class='danger'>Subject is suffering from an enlarged liver.</span>" //i.e. shrink their liver or give them a transplant.
+
+			//HEART
+			if(ishuman(M) && (istype(O, /obj/item/organ/heart)))
+				var/obj/item/organ/heart/He = O
+				if(H.undergoing_cardiac_arrest() && H.stat != DEAD)
+					temp_message += " <span class='danger'>Subject suffering from heart attack: Apply defibrillation or other electric shock <b>immediately!</b></span>"
+				if(He.organ_flags & ORGAN_FAILING)
+					heart_ded = TRUE
+
+			//TONGUE
+			if(istype(O, /obj/item/organ/tongue))
+				var/obj/item/organ/tongue/T = O
+				if(T.name == "fluffy tongue")
+					temp_message += " <span class='danger'>Subject is suffering from a fluffified tongue. Suggested cure: Yamerol or a tongue transplant.</span>"
+
+			//HECK
+			if(istype(O, /obj/item/organ/genital/penis))
+				var/obj/item/organ/genital/penis/P = O
+				if(P.length>20)
+					temp_message += " <span class='info'>Subject has a sizeable gentleman's organ at [P.length] inches.</span>"
+
+			if(istype(O, /obj/item/organ/genital/breasts))
+				var/obj/item/organ/genital/breasts/Br = O
+				if(Br.cached_size>5)
+					temp_message += " <span class='info'>Subject has a sizeable bosom with a [Br.size] cup.</span>"
+
+
+
+			//GENERAL HANDLER
+			if(!damage_message)
+				if(O.organ_flags & ORGAN_FAILING)
+					damage_message += " <span class='alert'><b>End Stage [O.name] failure detected.</b></span>"
+				else if(O.damage > O.high_threshold)
+					damage_message += " <span class='alert'>Chronic [O.name] failure detected.</span>"
+				else if(O.damage > O.low_threshold && advanced)
+					damage_message += " <font color='red'>Acute [O.name] failure detected.</span>"
+
+			if(temp_message || damage_message)
+				msg += "<b><span class='info'>[uppertext(O.name)]</b></span>: [damage_message] [temp_message]\n"
+
+
+
+		//END; LOOK FOR MISSING ORGANS?
+		if(!M.getorgan(/obj/item/organ/eyes))
+			msg += "\t<span class='alert'>Subject does not have eyes.</span>\n"
+		if(!M.getorgan(/obj/item/organ/ears))
+			msg += "\t<span class='alert'>Subject does not have ears.</span>\n"
+		if(!M.getorgan(/obj/item/organ/brain))
+			msg += "\t<span class='alert'>Subject's brain function is non-existent.</span>\n"
+		if(!M.getorgan(/obj/item/organ/liver))
+			msg += "\t<span class='alert'>Subject's liver is missing.</span>\n"
+		if(!M.getorgan(/obj/item/organ/lungs))

```

Comment:
FIY, xenos are carbons yet don't have lungs or liver (and probably more if it's still a larva). I'm not going to pretend you to fix it right now but an organ helper proc for this issue should be tailored in the future.

Reply:
I just realized we have getorgan and getorganslot as well.

Reply:
I've added a few things for it